src/Behat/GuzzleExtension/Context/GuzzleContext.php

```diff
@@ -268,6 +268,30 @@ public function iGetAnUnsuccessfulResponseWithAStatusCodeOf($code)
     }
 
     /**
+     * Checks response body matches the following value
+     *
+     * Example: The the response body matches the following:
+     *   """
+     *   Id,Name,Age,Comment
+     *   1,"Richard Saunders",33,"some comment"
+     *   2,"Dave Nash",,"another comment"
+     *   3,{stored[person][name]},{stored[person][age]},{stored[person][comment]}
+     *   """
+     *
+     * @Then the response body matches the following:
```

Comment:
Place `@Then` after PHP docblock directives, see other methods in this file for correct ordering.